cmd/internal/shared/applicationserver/config.go

```diff
@@ -46,4 +46,8 @@ var DefaultApplicationServerConfig = applicationserver.Config{
 		Workers:   16,
 		Downlinks: web.DownlinksConfig{PublicAddress: shared.DefaultPublicURL + "/api/v3"},
 	},
+	DeviceCache: applicationserver.DeviceCacheConfig{
+		Backend: "memory",
+		TTL:     time.Hour,
```

Comment:
This is very large and will confuse people. I'd go with something like 5 minutes.